senior-knights/course-schedulizer
# finish for video demo: harmony import

This PR was already shown in the Course Schedulizer demo. Now I'd like to merge it into dev. 

This PR adds:

- Updates Immer (closes #193)
- infers (some) assignments from an already imported schedule
- will submit fields when clicking next in the stepper (slow, but better UX than clicking)
- adds correct values to forms on the harmony stepper
- some cleanup for the demo

To test:

- `npm i && npm start`
- upload the cs schedule
- go to harmony
- click next
- click import
- click next until a schedule is found
- click finish
- click add the schedulizer
- see completed schedule

closes #136
